src/includes/before-breadcrumb/android.mdx

```diff
@@ -0,0 +1,13 @@
+```java
+import io.sentry.android.core.SentryAndroid;
```

Comment:
`m`: IMO we don't need to specify the import statements in the docs.

Reply:
I'm not a big fan either, but its being the pattern so far

Reply:
This is to avoid issues like this: https://github.com/getsentry/sentry-dart/issues/87

Reply:
It's part of our philosophy too: https://develop.sentry.dev/sdk/philosophy/#assume-novice-developers

Reply:
Hm, then I need to add this to Apple.